### DISCUSSION TITLE
Japanese transcription: remove silence works well?

### DISCUSSION THREAD

**Discussion by koka0630** [openai/whisper#108] [Q&A] [answered]
i tried transcription on the below condition;
file A(.wav): Japanese conversation 25min after 5min silence 
file B(.wav): the same Japanese conversation 25min (= removed 5min silence from file A)
The output transcription for file A was very poor. On the other hand, the output for file B gave me very good results.

Why does removing silence work well? 
though [README.md](https://github.com/openai/whisper#readme) says 'performing autoregressive sequence-to-sequence predictions **on each window.**'

my script is below
```
model = whisper.load_model("large")
args = {
    "verbose": True,
    "task": "transcribe",
    "language": "ja",
    "best_of": 5,
    "beam_size": 5,
    "patience": 0.0,
    "length_penalty": None,
    "suppress_tokens": "-1",
    "fp16": True,
}
result = whisper.transcribe(
    model,
    input_file,
    **args,
)
```

**Comment by shirayu** [comment, marked as answer]
FYI: This may be related to #29.

**Reply by koka0630** [reply]
thanks!!
i'll check it!
